### DISCUSSION TITLE
Notification Problem (no e-mails)

### DISCUSSION THREAD

**Discussion by jawoolJorgeSanchez** [globaleaks/globaleaks-whistleblowing-software#3404] [Q&A]
Hello I'm having problems with the auth.I'm not able to send any e-mail because we probably can do TLS auth. We changed the SMTP server and also the e-mail address. We are using version 4.11.0. How can we use the TLS auth with our exchange server

**Comment by evilaliv3** [comment]
Thank you for the question.

Would you please clarify better about your configuration?

**Reply by jawoolJorgeSanchez** [reply]
![imagen](https://user-images.githubusercontent.com/129067648/229464349-5dd75672-0fda-4a3f-95f6-ebbcc6432767.png)
The server is working. The problem is that we are not receiving the emails.
Can u specify which information do u need?

**Comment by evilaliv3** [comment]
@jawoolJorgeSanchez: Is it possibly for you to attach an extract of the file /var/globaleaks/log/globaleaks.log in corrispondence of the SMTP connections?

There you should be able to find the errors that clarify what you may have configured wrong.
